docker-compose.yml

```
@@ -27,6 +31,9 @@ services:
     restart: always
     ports:
       - 5678:5678
+    labels:
+    - "traefik.enable=true"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)